*** ovnkube-node.yaml	Wed Jan 13 15:22:13 2021
--- ovnkube-node.yaml.mod	Wed Jan 13 15:34:55 2021
***************
*** 101,107 ****
          securityContext:
            runAsUser: 0
            privileged: true
!           
  
          terminationMessagePolicy: FallbackToLogsOnError
          volumeMounts:
--- 101,108 ----
          securityContext:
            runAsUser: 0
            privileged: true
!           capabilities:
!             add: ["NET_BIND_SERVICE", "NET_ADMIN", "NET_RAW", "SYS_RAWIO", "SYS_CHROOT", "SYS_ADMIN", "SYS_PTRACE"]
  
          terminationMessagePolicy: FallbackToLogsOnError
          volumeMounts:
